### DISCUSSION TITLE
Plotting Lateral & Longitudinal accelerations

### DISCUSSION THREAD

**Discussion by nicZox** [theOehrly/Fast-F1#308] [Q&A]
Hello everyone,
I have read on various discussions that it is possible to compute and do plotting of the G-Force during the round (although not accurately because of the sampling rate).
Can anyone share the code to do this on lateral and longitudinal accelerations?

Thanks

**Comment by nicZox** [comment]
Could someone please explain to me how to calculate lateral G-Force? 
Thanks

**Reply by theOehrly** [reply]
This just a general problem from kinematics. You want to calculate the lateral acceleration for the trajectory of a generic particle. The Wikipedia article about kinematics will already give you an idea how to do that.

**Reply by nicZox** [reply]
> This just a general problem from kinematics. You want to calculate the lateral acceleration for the trajectory of a generic particle. The Wikipedia article about kinematics will already give you an idea how to do that.

Hi, thanks for the tip.
From what I cpaired to calculate the lateral acceleration, you need to know the radius of curvature of the cruves and circuits. I just didn't understand how to calculate them with Fast-F1.

Any advice?
Thanks

**Reply by Iskathor** [reply]
You need the real trajectory of the car, the information of the circuit is not enough
